### PR TITLE
Update ISSUE_TEMPLATE and PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by pubby**
I do not know if Github has changed how they parse markdown recently, but I do know that issue/PR directions have not been commenting out.

This changes the issue text to be commented out again, and also rewrites things slightly in hopes that it will lead to better submissions.

### CODE REVIEW

**Comment by pubby** [comment]
I removed the text about var-edits because its irrelevant most of the time and I think people would read+heed the directions more if they were kept concise.

I won't object to adding it back though.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1 @@
-[Note]: # (<B>Please enter the `Round ID` from the `Status` panel if you can!</B> If you believe the issue to be caused by a testmerge, please report it in its relative PR thread. State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))
-
-[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)
+[Note]: # (Include the Round ID from the Status panel or retrieve it from https://atlantaned.space/newSS13tools/round.php ! If you believe the issue to be caused by a testmerge, report it in the pull request's comment section instead. Explain your issue in detail, including whatever steps can be taken to reproduce it.)
```
Comment (Jordie0608) [inline .github/ISSUE_TEMPLATE.md]: I'd suggest a bit about how to check for testmerges i.e. Show-Server-Revision or the server notice.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -18,4 +17,4 @@ spellcheck: fixed a few typos
 experiment: added an experimental thingy
 /:cl:
 
-[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) 
+[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
```
Comment (Jordie0608) [inline .github/PULL_REQUEST_TEMPLATE.md]: Why two lines?

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (pubby) [reply .github/PULL_REQUEST_TEMPLATE.md]: Just to make it foolproof. With how lines wrap, sometimes people think they're writing on a new line when really they're not. When that happens, the comment breaks.

**Review by Jordie0608 — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1 @@
-[Note]: # (<B>Please enter the `Round ID` from the `Status` panel if you can!</B> If you believe the issue to be caused by a testmerge, please report it in its relative PR thread. State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))
-
-[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)
```
Comment (lzimann) [inline .github/ISSUE_TEMPLATE.md]: The admin part should stay

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**